src/auth/config.rs

```diff
@@ -56,71 +48,103 @@ fn is_self(scope_type: &str, scope: &Option<String>) -> Result<bool, String> {
 /// logged in but can make changes to any feature, including features created
 /// by another user
 ///
-fn is_auth(scope_type: &str, scope: &Option<String>) -> Result<bool, String> {
-    match scope {
-        &None => Ok(true),
-        &Some(ref scope_str) => {
-            match scope_str as &str {
-                "user" => Ok(true),
-                "admin" => Ok(true),
-                _ => Err(format!("Auth Config Error: '{}' must be one of 'user', 'admin', or null", scope_type)),
-            }
+fn is_auth(scope_type: &str, scope: &String) -> Result<bool, String> {
+    match scope.as_ref() {
+        "user" => Ok(true),
+        "admin" => Ok(true),
+        "disabled" => Ok(true),
+        _ => Err(format!("Auth Config Error: '{}' must be one of 'user', 'admin', or 'disabled'", scope_type)),
+    }
+}
+
+fn get_kv(scope: &str, key: &str, value: &serde_json::Value) -> Result<String, HecateError> {
+    match value.get(key) {
```

Comment:
`value` refers to both the value of a key/value object as well the object itself in this function. Can we clean that up a bit?